### PR TITLE
Contact us e2e

### PR DESCRIPTION
**Pull request by Diego416**
e2e testing for contact us page.

### CODE REVIEW

File: src/main/protractor/Tests/contactUs.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by juan muñoz on 30/06/17.
+ */
+
+describe('Contact us testing Web App', function() {
+
+    beforeEach(function() {
+        browser.get('https://test.govianex.com/');
```
Comment (lauraalvarezzz1) [inline src/main/protractor/Tests/contactUs.js]: we need to use before.all for the url

File: src/main/protractor/Tests/fastsendmoney.js [hunk 2]
```diff
@@ -30,173 +30,172 @@ describe('Fast send money Testing - Viamericas Web App', function() {
...
 });
 
-numbergenerator = function(min, max){
+numbergenerator = function(min, max) {
 
     return parseInt(Math.random() * (max - min) + min);
-};
-
-
+};
```
Comment (lauraalvarezzz1) [inline src/main/protractor/Tests/fastsendmoney.js]: Did you change any functionality of the fast send money flow?

File: src/main/protractor/po/homePage.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ var homePage = function() {
     this.logInButton = element(by.partialButtonText('Log In'));
     this.signUpButtonXpath = element(by.xpath('/html/body/div[2]/div/div[1]/div[1]/header/div/ul/li[3]/button[1]'));
     this.logInButtonXpath = element(by.xpath('/html/body/div[2]/div/div[1]/div[1]/header/div/ul/li[3]/button[2]'));
+    this.customerServiceButton = element(by.linkText('Customer Service'));
```
Comment (lauraalvarezzz1) [inline src/main/protractor/po/homePage.js]: Use xpath, we cannot use linkText because we have two languages

**Review by lauraalvarezzz1 — COMMENTED** [review]
Remember to solve the merge conflicts, you can do that here in github

File: src/main/protractor/Tests/fastsendmoney.js (hunk 2 above)
Comment (Diego416) [reply src/main/protractor/Tests/fastsendmoney.js]: No, I didn't

**Review by lauraalvarezzz1 — APPROVED** [review]
Done with the code review

**Pull request merged**